### PR TITLE
feat: add support for Terraform v0.14.x

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,9 +29,8 @@ ifndef NOCOLOR
 endif
 
 DOCKER_RUN_FLAGS += --rm
-DOCKER_RUN_FLAGS += -v ${PWD}:/app/src
+DOCKER_RUN_FLAGS += -v ${PWD}:/build
 DOCKER_RUN_FLAGS += -e TF_IN_AUTOMATION
-DOCKER_RUN_FLAGS += -e USER_UID=$(shell id -u)
```
Comment (mariux) [inline Makefile:37]: love it

File: Makefile [hunk 2]
```diff
@@ -49,27 +45,34 @@ DOCKER_RUN_CMD  = docker run ${DOCKER_FLAGS} ${BUILD_TOOLS_DOCKER_IMAGE}
 .PHONY: default
 default: help
 
-## Run pre-commit hooks in build-tools docker container.
+# Not exposed as a callable target by `make help`, since this is a one-time shot to simplify the development of this module.
+.PHONY: template/adjust
+template/adjust: FILTER = -path ./.git -prune -a -type f -o -type f -not -name Makefile
+template/adjust:
+	@find . $(FILTER) -exec sed -i -e "s,terraform-module-template,$${PWD##*/},g" {} \;
+
```
Comment (mariux) [inline Makefile:53]: not needed as the comment says it's a one-time thing... ;) i assume just copied so let's keep it ;)

**Review by mariux — APPROVED** [review]
lgtm

**Comment by mariux** [comment]
please always adjust pr titles after creating them with github cli..

File: Makefile (hunk 2 above)
Comment (soerenmartius) [reply Makefile:53]: yeah copy paste, didn't know whether we might need this later on. Let's keep it as it doesn't hurt :) I will try `git-xargs` later to get rid of it hehe

**Pull request merged**